intg/src/main/java/org/apache/atlas/model/lineage/AtlasLineageInfo.java

```diff
@@ -341,6 +345,14 @@ public void setHasDownstream(boolean hasDownstream) {
             this.hasDownstream = hasDownstream;
         }
 
+        public int getOffsetCount() {
```

Comment:
we are calling it "offsetCount" here and "from" in request payload - this will create confusion

Reply:
updated to getFromCounter